exercises/transpose/tests/output-2.txt

```diff
@@ -0,0 +1,16 @@
+TT
+hh
+ee
+
```

Comment:
Is this correct? I would expect that this line would have two spaces, as the original input also has two spaces.